### ISSUE TITLE
Class imported from third party library has type "Unknown"

### ISSUE THREAD

**Issue by Hubro** [microsoft/pyright#262]
**Describe the bug**

I'm not sure if this is a bug, but when I specify that the type of a variable is a class from a third party library, the type is set to `Unknown` instead. This makes static type checking completely useless.

**To Reproduce**

Install the `requests` library and put the following code into a Python file:

```python
import requests

foo: requests.Response = 123
```

**Expected behavior**

I would expect an error on the last line, saying something like:

> 'int' is incompatible with 'Response'

Instead the last line doesn't raise any errors.

![image](https://user-images.githubusercontent.com/597206/65532775-b7e6ec80-defc-11e9-9a4b-9b85b9cf8ed9.png) ![image](https://user-images.githubusercontent.com/597206/65532781-ba494680-defc-11e9-8845-b8b7c5733bdb.png)

If I use a local class (rather than one from a third party library) it works as expected:

![image](https://user-images.githubusercontent.com/597206/65532969-1d3add80-defd-11e9-9a76-899af3e27689.png)

Is this intended behavior? Is there anything I can do to remedy this?

**VS Code extension or command-line**

I am running pyright in VSCode. The plugin version is 1.0.65.

**Comment by erictraut** [comment]
Yes, this is the intended behavior if you don't have type stub files for that third-party library.

For an explanation and tips for how to address this, see: https://github.com/microsoft/pyright/blob/master/docs/type-stubs.md
